src/app/models/defectAdditionalInformation.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
do we want to put these in a models/generated/ folder?

Reply:
I think eventually we should have the models in along with the API services in an API folder but I'm still waiting to see what we find out about these swagger docs.